app/src/test/java/org/oppia/android/app/translation/AppLanguageWatcherMixinTest.kt

```diff
@@ -217,6 +216,28 @@ class AppLanguageWatcherMixinTest {
     assertThat(testActivityRecreator.getRecreateCount()).isEqualTo(1)
   }
 
+  @Test
+  fun testMixin_initialized_withShouldUseSystemLanguage_initializesSystemLanguage() {
+    val mixin = retrieveAppLanguageWatcherMixin()
+    mixin.initialize(shouldUseSystemLanguage = true)
+    testCoroutineDispatchers.runCurrent()
+
+    val localeContext = appLanguageLocaleHandler.getDisplayLocale().localeContext
+    assertThat(localeContext.languageDefinition.language).isEqualTo(ENGLISH)
+  }
+
+  @Test
+  fun testMixin_initialized_withoutShouldUseSystemLanguage_initializesSelectedLanguage() {
+    val mixin = retrieveAppLanguageWatcherMixin()
+    mixin.initialize(shouldUseSystemLanguage = false)
+    testCoroutineDispatchers.runCurrent()
+
+    updateAppLanguageTo(BRAZILIAN_PORTUGUESE)
```

Comment:
Shouldn't the test above include this update line? Otherwise it's not clear that the mixin is ignoring language changes in that configuration.

Reply:
i have updated coverages, thanks.